### PR TITLE
BLOOM minor fixes small test

### PR DESCRIPTION
**Pull request by younesbelkada**
Small modifications

- Modified docstring on tests
- Added correct revision on 350m model
- removed right padding left padding test

cc @ydshieh @NouamaneTazi @Muennighoff

### CODE REVIEW

File: tests/models/bloom/test_modeling_bloom.py [hunk 1]
```diff
@@ -396,7 +397,7 @@ def test_simple_generation(self):
         # >=760m + allow_fp16_reduced_precision_reduction = False  + torch.bmm  ==> PASS
 
         path_350m = "bigscience/bloom-350m"
-        model = BloomForCausalLM.from_pretrained(path_350m, use_cache=True).cuda()
+        model = BloomForCausalLM.from_pretrained(path_350m, use_cache=True, revision="gs555750").cuda()
```
Comment (NouamaneTazi) [inline tests/models/bloom/test_modeling_bloom.py:400]: What does the `revision` argument do? Are we sure we want to add that?

Comment (younesbelkada) [reply tests/models/bloom/test_modeling_bloom.py:400]: yes, this argument loads the model from the initial model we were using when designing those tests: https://huggingface.co/bigscience/bloom-350m/tree/gs555750 
@Muennighoff recently wanted to push the newest weights of 350m so just adding the revision flag should prevent us breaking those tests

File: tests/models/bloom/test_modeling_bloom.py [hunk 2]
```diff
@@ -461,33 +462,6 @@ def test_batch_generation_padd(self):
             tokenizer.decode(greedy_output_without_pad[0, :-3], skip_special_tokens=True),
         )
 
-    @slow
```
Comment (sgugger) [inline tests/models/bloom/test_modeling_bloom.py:464]: Could you just explain why this test is removed please?

**Review by sgugger — COMMENTED** [review]
Thanks for the fixes!

File: tests/models/bloom/test_modeling_bloom.py (hunk 2 above)
Comment (sgugger) [reply tests/models/bloom/test_modeling_bloom.py:464]: Thanks for clarifying!

**Review by sgugger — APPROVED** [review]

**Pull request merged**